packages/frontend/core/src/modules/quicksearch/providers/quick-search-provider.ts

```
@@ -8,7 +8,7 @@ export type QuickSearchFunction<S, P> = (
 
 export interface QuickSearchSession<S, P> {
   items$: LiveData<QuickSearchItem<S, P>[]>;
-  isError$?: LiveData<boolean>;
+  error$?: LiveData<any>;
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)